obs/project.go

```diff
@@ -91,3 +97,104 @@ type RepositoryPath struct {
 	Project    string `json:"project" xml:"project,attr"`
 	Repository string `json:"repository" xml:"repository,attr"`
 }
+
+func (c *Client) CreateUpdateProject(project *Project, ctx context.Context) error {
```

Comment:
Context must be the first parameter (lint error)

Reply:
Resolved